### PR TITLE
Showcase: Added SurveyEase

### PR DESCRIPTION
**Pull request by awazgyawali**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [X] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [X] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by jennifertrin** [comment]
Hello @awazgyawali Can you please advise if SurveyEase is live yet?

**Comment by sa-github-api** [comment]
Dear @awazgyawali,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/370) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: showcase.json
Comment (jennifertrin) [inline showcase.json:2671]: Can you please also add your Submittable ID?

File: showcase.json
Comment (jennifertrin) [inline showcase.json:2662]: Do you have another link that provides an overview of the product?

**Review by jennifertrin — CHANGES_REQUESTED** [review]

**Comment by awazgyawali** [comment]
> Hello @awazgyawali Can you please advise if SurveyEase is live yet?

No, we are still in the alpha state. We have a lot more to build to be able to take the product to market.

File: showcase.json
Comment (awazgyawali) [reply showcase.json:2662]: Unfortunately, this is what we have at the moment. Since the product is not in the state to be live, we havent focused on marketing contents.

File: showcase.json
Comment (awazgyawali) [reply showcase.json:2672]: Done @jennifertrin

**Comment by jennifertrin** [comment]
> > Hello @awazgyawali Can you please advise if SurveyEase is live yet?
> 
> No, we are still in the alpha state. We have a lot more to build to be able to take the product to market.

Could we wait until the marketing page is built out before we add to the ecosystem page?

**Comment by jennifertrin** [comment]
@awazgyawali Can you please create a new PR once you have launched and updated your landing page?

**Pull request closed without merging**
